src/applications/vaos/components/layout/DetailPageLayout.jsx

```diff
@@ -202,31 +212,176 @@ function CancelButton({ appointment }) {
+  isPastAppointment,
+}) {
+  if (!isPastAppointment) return null;
+  if (!claimData) return null;
+
+  let body = null;
+
+  if (
+    claimData.message === TRAVEL_CLAIM_MESSAGES.noClaim &&
+    daysRemainingToFileClaim > 0
+  ) {
+    body = (
+      <>
+        <p className="vads-u-margin-y--0p5">
+          Days left to file: {daysRemainingToFileClaim}
+        </p>
+        <p className="vads-u-margin-y--0p5">
+          <va-link
+            data-testid="file-claim-link"
+            className="vads-u-margin-y--0p5"
+            href={`/appointments/claims/?date=${appointmentDate}`}
+            text="File a travel reimbursement claim"
+          />
+        </p>
+      </>
+    );
+  }
+  if (
+    claimData.message === TRAVEL_CLAIM_MESSAGES.noClaim &&
+    daysRemainingToFileClaim < 1
+  ) {
+    body = (
+      <>
+        <p className="vads-u-margin-y--0p5">
+          Days left to file: {daysRemainingToFileClaim}
+        </p>
+        <p className="vads-u-margin-y--0p5">
+          You didn’t file a claim for this appointment. You can only file for
+          reimbursement within 30 days of the appointment.
+        </p>
+        <p className="vads-u-margin-y--0p5">
+          <va-link
+            data-testid="how-to-file-claim-link"
+            className="vads-u-margin-y--0p5"
+            href="https://www.va.gov/resources/how-to-file-a-va-travel-reimbursement-claim-online/"
+            text="Learn more about travel reimbursement"
+          />
+        </p>
+      </>
+    );
+  }
+  if (claimData.id) {
+    body = (
+      <>
+        <p className="vads-u-margin-y--0p5">
+          You've already filed a claim for this facility and date.
+        </p>
+        <p className="vads-u-margin-y--0p5">
+          <va-link
+            data-testid="view-claim-link"
+            href={`/appointments/claims/${claimData.id}`}
+            text="Check your claim status"
+          />
+        </p>
+      </>
+    );
+  }
+
+  if (!body) return null;
+
+  return <Section heading="Travel reimbursement">{body}</Section>;
+}
+
+TravelReimbursement.propTypes = {
+  appointmentDate: PropTypes.string,
+  claimData: PropTypes.object,
+  daysRemainingToFileClaim: PropTypes.number,
+  isPastAppointment: PropTypes.bool,
+};
+
+export function AppointmentTasks({
+  appointmentDate,
+  claimData,
+  daysRemainingToFileClaim,
+  isPastAppointment,
+}) {
+  if (!isPastAppointment) return null;
+  if (!appointmentDate) return null;
+  if (!claimData) return null;
+  if (claimData.message !== TRAVEL_CLAIM_MESSAGES.noClaim) return null;
+  if (daysRemainingToFileClaim < 1) return null;
+  return (
+    <Section heading="Appointment tasks">
+      <va-link-action
+        data-testid="file-claim-link"
+        className="vads-u-margin-top--1"
+        href={`/appointments/claims/?date=${appointmentDate}`}
+        text="File a travel reimbursement claim"
+      />
+      <p className="vads-u-margin-top--0 vads-u-margin-bottom--1 action-link-below-text">
```

Comment:
Instead of a custom `action-link-below-text` class I think `vads-u-margin-left--4` works here. I believe for most `vads-u` classes the conversion is 1 = 0.5rem.

Reply:
huh I thought I tried that and it was slightly off but it works thanks!